ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -77,6 +76,7 @@ export const TokenListItem = ({
   isNativeCurrency = false,
   isStakeable = false,
   address = null,
+  showPercentage = true,
```

Comment:
Let's default to false here to avoid having to explicitly set it to false when not used